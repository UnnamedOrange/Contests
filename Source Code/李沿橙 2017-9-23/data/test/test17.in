5
49
971 11 3 15 910 2 867 10 9 901 3 982 4 2 6 9 6 8 11 6 11 892 944 851 890 10 940 3 989 872 923 964 13 8 10 11 930 868 902 953 11 4 964 863 952 10 2 921 2 
40
128 808 178 383 590 722 785 431 296 294 674 982 946 786 776 669 75 972 806 208 138 641 625 420 690 206 922 797 611 805 630 387 297 269 78 782 916 65 887 798 
36
996 968 434 674 305 354 233 958 371 680 967 78 569 266 93 315 243 99 369 335 331 601 192 999 364 685 216 442 999 649 374 962 629 263 38 277 
50
294 861 711 959 315 122 933 879 831 781 671 781 545 975 578 684 972 288 589 536 396 882 860 990 657 81 439 494 767 127 565 717 521 335 897 999 572 707 600 709 481 658 312 697 856 310 286 481 643 746 
17
48 96 4 11 76 72 27 37 913 58 32 689 923 67 17 5 32 
