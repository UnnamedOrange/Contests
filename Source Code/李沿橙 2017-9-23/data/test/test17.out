2 2 2 2 3 3 3 4 4 6 6 6 8 8 9 9 10 10 10 10 11 11 11 11 11 13 851 867 868 872 890 892 901 902 910 921 923 930 940 944 952 953 964 964 971 982 989 863 15 
65 75 78 128 138 178 206 269 294 296 297 383 420 590 625 630 641 669 674 690 722 776 782 785 797 798 805 806 808 887 916 922 972 982 946 786 611 431 387 208 
38 78 93 99 192 216 233 243 263 266 277 305 331 335 354 364 369 371 374 434 442 601 629 649 674 680 958 967 968 996 999 999 962 685 569 315 
81 122 286 294 310 312 315 335 396 481 481 494 521 536 545 565 572 578 589 643 657 658 671 684 697 707 709 711 717 767 781 781 831 856 860 861 879 882 933 959 972 975 990 999 897 746 600 439 288 127 
4 5 11 17 27 32 32 37 48 58 67 72 76 96 913 923 689 
